app/cinema/hall.py

```diff
@@ -0,0 +1,11 @@
+class CinemaHall:
+    def __init__(self, number):
+        self.number = number
+
+    def movie_session(self, movie_name: str, customers: list, cleaning_staff):
+        print(f"\"{movie_name}\" started in hall number {self.number}.")

```

Comment:
Taras is right. Use:
```suggestion
        print(f'"{movie_name}" started in hall number {self.number}.')

```